VVResonances/interactive/makeCard.py

```diff
@@ -5,7 +5,9 @@
 cmd='combineCards.py '
 
 purities=['HPHP','HPLP','LPLP']
-purities=['HPLP']
+label="testBinning_"
```

Comment:
I would remove this "label" and everything where it is used since it is only for your local testing.